ansible/pmm2/post-build-actions.yml

```diff
@@ -25,11 +25,20 @@
 
     # TODO https://jira.percona.com/browse/PMM-4991
     - name: Detect distribution        | Create '/srv/pmm-distribution' file for AMI
-      when: ansible_virtualization_type == "xen" or ansible_virtualization_type == "kvm"
+      when: >
+        ( ansible_virtualization_type == "xen"
+        or ansible_virtualization_type == "kvm" )
+        and ansible_system_vendor != "DigitalOcean"
       copy:
         content: "ami"
         dest: /srv/pmm-distribution
 
+    - name: Detect distribution        | Create '/srv/pmm-distribution' file for DigitalOcean
+      when: ansible_system_vendor == "DigitalOcean"
+      copy:
+        content: "digitalocean"
```

Comment:
That will break telemetry reporting: https://github.com/percona/pmm-managed/blob/7095c82d355897a819b146ffa71890a8d5ae7152/services/telemetry/telemetry.go#L124-L144

Reply:
@AlekSi - would it be possible to add a case to that `getDistributionMethodAndOS` method for `digitalocean`? I'm just looking at the changes that have been made in this repo since I made this PR, and it looks like `/srv/pmm-distribution` isn't even created anymore, so maybe this is no longer relevant. If it is relevant, then none of the `ami/ovf/docker` options fit the DigitalOcean deployment case.